### DISCUSSION TITLE
guidance on associating an authorised account to add a social account for an existing user

### DISCUSSION THREAD

**Discussion by johndpope** [supabase/supabase#8897] [Questions] [answered]
https://github.com/netlify/gotrue/issues/325

we have users - they sign up using facebook -
now we want to have them associate their account with their google / youtube etc.
is there existing flows to handle this?

I just imagine it being another entry into identities - or ????

From my testing - it's clear if I use email signup - then **separately** sign up using google with **same** email - the identities are collected together - seamlessly stored into one account.
But what if they are not using same email? how can I avoid redoing sign up - and just associate an existing account with an authorized account...

**Comment by GaryAustin1** [comment, marked as answer]
Currently an email is required and must be the same to tie email/3rd party auths.
This is relevant on the email requirement and where not requiring it is at: https://github.com/supabase/gotrue/pull/414
